charts/opentelemetry-kube-stack/README.md

```diff
@@ -1,8 +1,11 @@
 # OpenTelemetry Kube Stack Helm Chart
 
-> [!CAUTION]
-> This chart is under active development and is not meant to be installed yet.
-> Right now, nothing is included with this deployment.
+| Status        |           |
```

Comment:
@TylerHelmuth i copied what the collector has to show that this in beta and how to open issues